internal/app/client/client.go

```diff
@@ -143,13 +143,42 @@ func invoke(ctx context.Context, req *v1alpha1.ClientCompatRequest) (*v1alpha1.C
 		clientOptions = append(clientOptions, connect.WithProtoJSON())
 	}
 
-	// TODO - Add support for other compression algos
 	switch req.Compression {
+	case v1alpha1.Compression_COMPRESSION_BR:
+		clientOptions = append(
+			clientOptions,
+			connect.WithAcceptCompression(
+				compression.Brotli,
+				compression.NewBrotliDecompressor,
+				compression.NewBrotliCompressor,
+			),
+		)
+		clientOptions = append(clientOptions, connect.WithSendCompression(compression.Brotli))
```

Comment:
Stylistically, I think this might be better as a single append, instead of two:
```suggestion
		clientOptions = append(
			clientOptions,
			connect.WithAcceptCompression(
				compression.Brotli,
				compression.NewBrotliDecompressor,
				compression.NewBrotliCompressor,
			),
			connect.WithSendCompression(compression.Brotli),
		)
```

Reply:
Good call. Updated.